### ISSUE TITLE
PFQuery Failing Always

### ISSUE THREAD

**Issue by Ponyboy47** [parse-community/Parse-SDK-iOS-OSX#360]
I'm not sure why this is happening, but every time I try any PFQuery it fails with no error message and returns nothing.

I'm using Xcode 7.0 and writing a Swift 2.0 application on OS X El Capitan.

If I do the query on the main thread then Xcode prints this error:
`Error Domain=Foundation._GenericObjCError Code=0 "(null)"`

I've tried querying multiple different tables, I've done empty queries to return all objects, and I can successfully write to each of these tables. For whatever reason though, querying doesn't work and always returns nothing.

**Comment by richardjrossiii** [comment]
I think there's something more going on here. I just tested on Xcode 7.0.1 (7A1001) running on El Capitan Build 15B22c, and everything works swimmingly for me.

I would suggest making sure that your SDK is up-to-date (latest is 1.8.5), and try again.

If that still doesn't work, I encourage you to download this test project, and see if it works for you, and if it does not, it's probably a larger issue with your OS X setup (maybe a firewall).

https://github.com/richardjrossiii/ParseSwiftTest

If the test project does work, then it's most likely an issue with your usage of Parse inside of your application, and I would recommend you post some excepts of how you're using the failing queries here.

**Comment by Ponyboy47** [comment]
The test project works just fine and I ensured I am using the latest SDK (version 1.8.5)

I am familiar with the Parse API because I've developed apps using the iOS version of the SDK before and have never had any issues.

In my OS X app I parse a CSV file (of roughly 8k entries) and create/save these objects to my Parse database. I have successfully used the subclassing protocols to create my own object that I use to write to my database, but for whatever reason querying fails. I've even gone down to just a basic query and it still fails. Here is the exact query I do and the error it always returns:
<pre><code>let query = School.query()!
// let query = PFQuery(className: "School")
do {
    let objs = try query.findObjects()
    print(objs)
} catch let error as NSError {
    print(error)
}
</code></pre>

The above code fails regardless of whether I use the subclassed School query object or the standard PFQuery with the class name. Every time the error returned is:
`Error Domain=Foundation._GenericObjCError Code=0 "(null)"`

I just went in to System Preferences and completely turned off my Firewall, but I still had the exact same issue as well..

What other steps may I do to troubleshoot or help you to reproduce this issue?
